zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/data/Size3D.kt

```diff
@@ -60,7 +60,7 @@ data class Size3D private constructor(val xLength: Int,
      */
     fun containsPosition(position: Position3D): Boolean {
         val (x, y, z) = position
-        return xLength > x && yLength > y && zLength > z
+        return x in 0 until xLength && y in 0 until yLength && z in 0 until zLength
```

Comment:
This will have an asymptotic complexity of `O(n)` where `n` is equal to `xLength` and `yLength`. I'd do this with a simple bounds check instead (less then, greater then) which has the complexity of `O(1)`.

Reply:
You're right - that's really something I didn't think of. Changed it already

Reply:
👍